.github/workflows/publish-and-deploy.yaml

```diff
@@ -43,3 +43,21 @@ jobs:
     secrets: inherit
     needs: push_to_registry_prod
     if: github.ref == 'refs/heads/main'
+  
+  push_to_registry_prod2:
+    name: Push to prod2
+    uses: ./.github/workflows/build-push-image.yaml
+    with:
+      environment: prod2
+    secrets: inherit
+    needs: deploy_to_dev3
+    if: github.ref == 'refs/heads/main'
```

Comment:
I'm not sure we need to do this anymore. The only reason we did in the past was because we used to have client side environment variables, but we don't anymore. What do you think about making a centralized image?

Reply:
Good call, please recheck

Reply:
https://github.com/nycmeshnet/meshforms/issues/124